### PR TITLE
Add support for conafigurable userinfo_endpoint and a better error handling when this is missing

### PR DESCRIPTION
**Pull request by wiliamsouza**
Add support for configurable userinfo_endpoint and a better error handling when OpenID connect provider discovery metadata does not contain this field.

### CODE REVIEW

File: oidc_auth/authentication.py [hunk 1]
```diff
@@ -68,7 +68,11 @@ def get_bearer_token(self, request):
 
     @cache(ttl=api_settings.OIDC_BEARER_TOKEN_EXPIRATION_TIME)
     def get_userinfo(self, token):
-        response = requests.get(self.oidc_config['userinfo_endpoint'],
+        userinfo_endpoint = self.oidc_config.get('userinfo_endpoint', api_settings.USERINFO_ENDPOINT)
+        if userinfo_endpoint in ['', None, [], (), {}]:
```
Comment (allardhoeve) [inline oidc_auth/authentication.py]: I would rather this be `if not userinfo_endpoint:`, which is much more readable and probably what you want anyway.

**Comment by wiliamsouza** [comment]
@allardhoeve Can you please take a look here.

**Comment by kerrermanisNL** [comment]
Looks pretty good @wiliamsouza do you have time to rebase?

**Comment by rburgst** [comment]
I would also need this feature. Any chance we can get this merged?

**Comment by kerrermanisNL** [comment]
Closing in favour of #54 thanks again @wiliamsouza :)

**Pull request closed without merging**